TrackletAlgorithm/Constants.h

```diff
@@ -109,6 +110,8 @@ constexpr double VMROUTERCUTRD1D3 = 55.0;    //Max r for disk seeds in cm
 // various bit widths
 constexpr unsigned nbitsallstubs[N_LAYER + N_DISK] = {3, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2};
 constexpr unsigned int nbits_maxvm = 5; // number of bits needed for max number of VMs per layer/disk (max number is 32)
+// number of bits used to distinguish VMs in one allstub block for each layer
+constexpr unsigned int nbits_vmmelayers[6]={2,3,3,3,3,3};
```

Comment:
Why were only nbits_vmmelayers added, and not nbits_vmmedisks as well? Also, do we want to put all the nbits_vmme numbers in one single array like what is done for nbitsallstubs?